tests/tests_geomstats/test_geometry/test_spd_matrices.py

```diff
@@ -2,49 +2,105 @@
 
 import pytest
 
+import geomstats.backend as gs
+from geomstats.geometry.positive_lower_triangular_matrices import (
+    PositiveLowerTriangularMatrices,
+)
+from geomstats.geometry.scalar_product_metric import ScalarProductMetric
 from geomstats.geometry.spd_matrices import (
+    CholeskyMap,
+    MatrixPower,
     SPDAffineMetric,
     SPDBuresWassersteinMetric,
     SPDEuclideanMetric,
     SPDLogEuclideanMetric,
     SPDMatrices,
+    SPDPowerMetric,
+    SymMatrixLog,
 )
+from geomstats.geometry.symmetric_matrices import SymmetricMatrices
 from geomstats.test.parametrizers import DataBasedParametrizer
 from geomstats.test.random import RandomDataGenerator
+from geomstats.test_cases.geometry.diffeo import DiffeoTestCase
+from geomstats.test_cases.geometry.pullback_metric import PullbackDiffeoMetricTestCase
 from geomstats.test_cases.geometry.riemannian_metric import RiemannianMetricTestCase
 from geomstats.test_cases.geometry.spd_matrices import (
     SPDEuclideanMetricTestCase,
     SPDMatricesTestCase,
 )
 
+from .data.diffeo import DiffeoTestData
 from .data.spd_matrices import (
-    SPD2AffineMetricPower1TestData,
+    CholeskyMapSmokeTestData,
+    MatrixPower05TestData,
+    SPD2AffineMetricTestData,
     SPD2BuresWassersteinMetricTestData,
-    SPD2EuclideanMetricPower1TestData,
+    SPD2EuclideanMetricTestData,
     SPD2LogEuclideanMetricTestData,
     SPD3AffineMetricPower05TestData,
     SPD3BuresWassersteinMetricTestData,
-    SPD3EuclideanMetricPower1TestData,
     SPD3EuclideanMetricPower05TestData,
+    SPD3EuclideanMetricTestData,
     SPD3LogEuclideanMetricTestData,
-    SPDAffineMetricPower1TestData,
     SPDAffineMetricTestData,
     SPDBuresWassersteinMetricTestData,
     SPDEuclideanMetricTestData,
     SPDLogEuclideanMetricTestData,
     SPDMatrices2TestData,
     SPDMatrices3TestData,
     SPDMatricesTestData,
+    SymMatrixLogSmokeTestData,
 )
 
 # TODO: check affine invariance
+# TODO: add SPDPowerMetricTestCase against some power=1?
```

Comment:
Put TODOs in github issues, not in the code (otherwise the nb of TODOs  will blow up)

Reply:
Done.